discovery-infra/test_infra/config/__init__.py

```diff
@@ -0,0 +1,9 @@
+from ._config import EnvConfig
+from ._cluster_config import DefaultClusterConfig, DefaultInstallConfig
```

Comment:
although possible, it's not very convenient setting private modules
so I would suggest rename ``_<module>`` to just ``<module>``

Reply:
I think it's a good practice to control the each module accessibility and not give access to all module ant it's objects

Reply:
@eliorerz i don't understand why it is needed